additional_rule_metadata.json

```diff
@@ -744,6 +783,14 @@
         "recommendation": "true,",
         "level": "Opportunity for Excellence"
     },
+    "increment-decrement": {
+        "issueClass": "Non-SDL",
+        "issueType": "Warning",
+        "severity": "Moderate",
```

Comment:
`severity` and `commonWeaknessEnumeration` differs from initial [rule metadata](https://github.com/Microsoft/tslint-microsoft-contrib/blob/9b25025c4912de65ca869c93a37c8bec84c94111/src/noIncrementDecrementRule.ts#L30-L33)

Is it intentional change?